build-system/tasks/presubmit-checks.js

```diff
@@ -566,6 +566,7 @@ var forbiddenTermsSrcInclusive = {
   // Functions
   '\\.changeHeight\\(': bannedTermsHelpString,
   '\\.changeSize\\(': bannedTermsHelpString,
+  '\\.attemptChangeHeight\\(0\\)': 'please consider using `attemptCollapse()`',
```

Comment:
Why not use a `dev().assert`? We could catch both `height = 0` and `width = 0` changes then.